### DISCUSSION TITLE
In-App user Analytics & Statistics

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#33] [Ideas & Features]
I think that users should have the ability to view their own statistics, as well as aggregated information about the people around them so that they can see what is going on. Things like how many people are in their area that meets their filters, how many are there in total, what the male to female ratio is, how many people are active, what percentage of people swipe on them, what percentage of people they swipe on, how many users they have been shown to etc.

**Comment by ajbott** [comment]
Provide users with some of the following options when working with and viewing data.

## Viewing Profile Stats - TRANSPARENCY

Users should have the ability to access comprehensive statistics about their profiles and general aggregate data, promoting transparency and empowering them to make informed choices. This feature aims to provide insights that help users understand how people engage with their profiles and offers opportunities for improvement.

### Example Stats:

- **Profile views vs likes**
- **Likes per-picture**
- **Time spent viewing profile vs average**
- **Number of People in the Area Range:**
  This statistic informs users about the pool of potential matches in their specified geographic area, helping them understand the size of their dating pool.

- **Percentage of Matches Meeting Filter Criteria:**
  Indicates what percentage of potential matches align with a user's specific filter criteria, such as age, interests, or other attributes.

- **Total Number of People Who Have Swiped on You:**
  Provides insight into a user's popularity on the platform, counting people who have shown interest by swiping right or liking their profile.

- **Percentage of Matched Users Meeting Filter Criteria:**
  Similar to the previous point but focuses specifically on users with whom the user has matched, revealing the percentage of matches aligning with filter criteria.

- **Average Number of Messages per Conversation:**
  Indicates how engaging a user's conversations are by providing an average count of messages exchanged, reflecting the depth and quality of interactions.

- **Feedback from Past Matches:**
  Includes reviews, comments, or ratings from users who have interacted with the profile owner, helping users understand how others perceive them.

- **Percentage of Matches Leading to Dates:**
  Reveals the effectiveness of a user's profile and interactions by showing the percentage of matches that progressed to in-person meetings or dates.

## Provide Monthly Data Analytics

### Trending Data:

- **Most popular filters**
- **Number of matches made**
- **Number of relationships formed**
- **Newly created categories**
- **What most women in the area are looking for**
- **What most men are looking for**

### Discoveries:

- **People that X match with people with Y**

### Interesting Data Points:

- **Women moving down in height filter**

### Benefits:

This monthly analytics report allows users to:

- See what others are interested in
- Discover something that resonates with them
- Understand trends in the dating community
- Make informed decisions based on data-driven insights
